lib/Target/ExportSystemC/EmissionPrinter.cpp

```diff
@@ -89,7 +89,7 @@ InlineEmitter EmissionPrinter::getInlinable(Value value) {
       << value << "'\n";
   err.attachNote(requestLoc) << "requested to be inlined here";
   return InlineEmitter(
-      [&]() { os << "<<INVALID VALUE TO INLINE (" << value << ")>>"; },
+      [=]() { os << "<<INVALID VALUE TO INLINE (" << value << ")>>"; },
```

Comment:
Why is this changed to capture by copy?

Reply:
See the GCC build failure above. There was an issue with variadic result values such as `%0:3 = call ...`. I don't know exactly how this is handled in MLIR, but basically the reference to `value` was invalid at the point where this lambda is called.